### PR TITLE
Remove Stock Template(#93)

### PR DESCRIPTION
**Pull request by venkateshdb**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<93>`
- Preview Link : https://venkateshdb.github.io/gci18.fossasia.org/
-

previous
> ## After
now

- Removed Stock templates and added some UI changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ _site/
 .sass-cache/
 Gemfile.lock
 CNAME
+Thumbs.db
```
Comment (simsausaurabh) [inline .gitignore]: Add EOF.

File: _includes/header.html [hunk 2]
```diff
@@ -50,30 +51,14 @@
...
-              </li>
-            </ul>
-          </div>
-        </div>
-        <!--end of module group-->
-      </div>
     </nav>
   </div>
-  <div class="main-container">
+  <div class="main-container">
```
Comment (simsausaurabh) [inline _includes/header.html]: Add EOF

File: css/custom.css [hunk 3]
```diff
@@ -80,17 +80,27 @@ p.person-name {
...
 }
+*/
 
 .single-mentor:hover > img {
   filter: brightness(50%);
   transition: all .6s;
 }
 
+
+
```
Comment (simsausaurabh) [inline css/custom.css]: Please remove extra spaces.

File: .gitignore (hunk 1 above)
Comment (venkateshdb) [reply .gitignore]: what do you mean by End of file

File: css/custom.css (hunk 3 above)
Comment (venkateshdb) [reply css/custom.css]: should I minify the file

**Comment by simsausaurabh** [comment]
@venkateshdb Thanks for the code, it would really help if you could squash your commits into one.

**Comment by venkateshdb** [comment]
can you help me with squashing the commits.
I'm using 
> git rebase -i HEAD~n (n: no of commits to rebase) 

but I am getting merge conflicts, and when try to resolve it doesn't show the recent changes i made

**Comment by jogendra** [comment]
@venkateshdb Make sure when you run the command mentioned above, `n` should not include merge commits (I am not really sure but remember) and see the first commit is the first commit of this PR. Also after resolving conflicts, you may need to add changes. Just add the changes by `git add .` (DO NOT COMMIT). and then `git rebase --continue` to continue. hope this helps you

**Comment by venkateshdb** [comment]
When i am using rebase .I am also getting others people's  commit  message.

**Comment by fragm3** [comment]
@venkateshdb you can ignore other people commit by adding '#' in front of the commit. 
Useful link: https://davidwalsh.name/squash-commits-git

**Comment by venkateshdb** [comment]
I think this is ok @fragm3 @jogendra  @simsausaurabh 
, Thanks for helping , finally I understood and rebase's and how to deal with them

**Comment by jogendra** [comment]
@venkateshdb Great :) Please resolved conflicts

**Comment by venkateshdb** [comment]
Which one

**Comment by abishekvashok** [comment]
There are conflicts in this pr, please resolve them.
Reference:
https://www.git-tower.com/learn/git/ebook/en/command-line/advanced-topics/merge-conflicts

**Comment by venkateshdb** [comment]
Now is it ok @jogendra , @simsausaurabh  , @fragm3 , @abishekvashok

**Comment by sridharjajoo** [comment]
@venkateshdb Please resolve the conflicts

**Comment by venkateshdb** [comment]
I am closing this pull request as it's getting to complicated to fix merge conflicts but I will reopen it.

**Pull request closed without merging**